### PR TITLE
view course details

### PR DESCRIPTION
**Pull request by ghost**
image

### CODE REVIEW

**Comment by ghost** [comment]
redmine ticket https://edu-redmine.sun-asterisk.vn/issues/10613

**Comment by ghost** [comment]
ready

File: app/views/courses/_course.html.erb [hunk 1]
```diff
@@ -3,11 +3,16 @@
     <%= image_tag course.image, size: "360x220", alt: t(".course_image"), class: "course_img" %>
     <div class="hover_text">
       <a href="#">
-        <h4><%= course.name %></h4>
+        <h4><%= link_to course.name, course %></h4>
       </a>
       <ul class="list">
-        <li><a href="#"><i class="lnr lnr-users"></i> 355</a></li>
-        <%= render partial: "trainer", collection: course.users, as: :user %>
+        <li><a href="#"><i class="lnr lnr-users"></i> <%= course.users.count{|user| user.trainee?} %></a></li>
```
Comment (VuThiTranVan) [inline app/views/courses/_course.html.erb]: cho vào helper hì
`course.users.count{|user| user.trainee?} `

File: app/views/courses/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <div class="c_details_list">
+          <ul class="list">
+            <li>
+                <a href="#"><%= t".trainer" %>
+                    <span>
+                      <%= render partial: "trainer", collection: @course.users, as: :user %>
+                    </span>
+                </a>
+            </li>
+            <li><a href="#"><%= t".members" %> <span><%= @course.user_courses.count{|user_course| user_course.user.trainee?} %></span></a></li>
```
Comment (VuThiTranVan) [inline app/views/courses/show.html.erb]: ` @course.user_courses.count{|user_course| user_course.user.trainee?} `
viết vào hepler, đỡ lặp code

**Comment by ghost** [comment]
ready

File: app/views/courses/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          <ul class="list">
+            <li>
+                <a href="#"><%= t".trainer" %>
+                    <span>
+                      <%= render partial: "trainer", collection: @course.users, as: :user %>
+                    </span>
+                </a>
+            </li>
+            <li><a href="#"><%= t".members" %> <span><%= count_trainee_in @course %></span></a></li>
+            <li><a href="#"><%= t".schedule" %> <span><%= @course.start_date %> <%= t".to" %> <%= @course.end_date %></span></a></li>
```
Comment (VuThiTranVan) [inline app/views/courses/show.html.erb:50]: sử dụng format datetiem trong I18n nhé
`<%= @course.start_date %> <%= t".to" %> <%= @course.end_date %>`

**Pull request merged**